geomesa/src/main/scala/geotrellis/spark/io/geomesa/GeoMesaFeatureReader.scala

```diff
@@ -0,0 +1,54 @@
+package geotrellis.spark.io.geomesa
+
+import geotrellis.geotools._
+import geotrellis.spark._
+import geotrellis.vector._
+
+import org.apache.accumulo.core.client.mapreduce.InputFormatBase
+import org.apache.hadoop.io.Text
+import org.apache.hadoop.mapreduce.Job
+import org.apache.spark.rdd.RDD
+import org.apache.spark.SparkContext
+import org.geotools.data._
+import org.locationtech.geomesa.jobs.mapreduce.GeoMesaInputFormat
+import org.locationtech.geomesa.jobs.GeoMesaConfigurator
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+import scala.reflect.ClassTag
+
+class GeoMesaFeatureReader(val instance: GeoMesaInstance)(implicit sc: SparkContext) extends Serializable {
+  def readSimpleFeatures(
+    featureName: String,
+    query: Query,
+    simpleFeatureType: SimpleFeatureType,
+    numPartitions: Option[Int] = None
+  ): RDD[SimpleFeature] = {
+    val dataStore = instance.accumuloDataStore
+    dataStore.createSchema(simpleFeatureType)
```

Comment:
this is a no-op if the schema already exists - but if the schema doesn't exist then there's not going to be any data in it... maybe that is ok here.